components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -194,38 +209,40 @@ SUBROUTINE p3_init_b()
 
     !write(iulog,*) '   Generating rain lookup-table ...'
 
+    ! AaronDonahue: Switching to table ver 4 means switching to a constand mu_r,
+    ! so this section is commented out.
     do i = 1,150              ! loop over lookup table values
-       initlamr = 1./((real(i)*2.)*1.e-6 + 250.e-6)
-
-       ! iterate to get mu_r
-       ! mu_r-lambda relationship is from Cao et al. (2008), eq. (7)
-
-       ! start with first guess, mu_r = 0
-
-       mu_r = 0.
-
-       do ii=1,50
-          lamr = initlamr*((mu_r+3.)*(mu_r+2.)*(mu_r+1.)/6.)**thrd
-
-          ! new estimate for mu_r based on lambda
-          ! set max lambda in formula for mu_r to 20 mm-1, so Cao et al.
-          ! formula is not extrapolated beyond Cao et al. data range
-          dum  = min(20.,lamr*1.e-3)
-          mu_r = max(0.,-0.0201*dum**2+0.902*dum-1.718)
-
-          ! if lambda is converged within 0.1%, then exit loop
-          if (ii.ge.2) then
-             if (abs((lamold-lamr)/lamr).lt.0.001) goto 111
-          end if
-
-          lamold = lamr
-
-       enddo
-
-111    continue
-
-       ! assign lookup table values
-       mu_r_table(i) = mu_r
+!       initlamr = 1./((real(i)*2.)*1.e-6 + 250.e-6)
+!
```

Comment:
Any reason to keep this commented-out code?

Reply:
Not really, we are going to be issuing another PR shortly after this one that addresses all this commented block.  It's a change that Hugh's team put in that we still don't fully understand so I'm not ready to remove it just yet.

Reply:
ok